e2e/tests/localAuthoritySubmitsApplication_test.js

```diff
@@ -452,12 +557,14 @@ Scenario('local authority tries to submit without giving consent', async (I, cas
 });
 
 Scenario('local authority submits after giving consent @create-case-with-mandatory-sections-only', async (I, caseViewPage, submitApplicationEventPage) => {
-  await caseViewPage.goToNewActions(config.applicationActions.submitCase);
+  await caseViewPage.startTask(config.applicationActions.submitCase);
   // I.see('£2,055.00'); Disabled until Fee Register updated on AAT
```

Comment:
We should probably re-enable this at some point...

Reply:
yeah, but not here :)